charts/app-config/values-integration.yaml

```diff
@@ -975,15 +975,19 @@ govukApplications:
       arch: arm64
       rails:
         createKeyBaseSecret: false
+      redis:
+        enabled: true
+        redisUrlOverride:
+          app: &email-alert-draft-frontend-redis >
+            redis://shared-redis-govuk.eks.integration.govuk-internal.digital
+          workers: *email-alert-draft-frontend-redis
```

Comment:
This app doesn't have any workers (a value `workers: enabled: true`) in the helm chart would indicate the app having workers.

So each environment can be switched in one PR, with the change to just this and no draining required:
```yaml
      redis:
        enabled: true
```